controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -1529,9 +1529,8 @@ func init() {
 }
 
 func checkFinalizers(req *common.HcoRequest) bool {
-	finDropped := false
-
 	if hcoutil.ContainsString(req.Instance.ObjectMeta.Finalizers, badFinalizerName) {
+		finDropped := false
```

Comment:
This is a very old code, dealing with an old finalizer name, Do we still need to check it?

Reply:
yes, it's the time for that!